Sources/Bow/Data/Option.swift

```diff
@@ -278,9 +278,16 @@ extension Option: Semigroup where A: Semigroup {
 
 // MARK: Instance of `Semigroupal` for `Option`,
 extension ForOption: Semigroupal {
-	public static func product<A, B>(_ a: Kind<ForOption, A>, _ b: Kind<ForOption, B>) -> Kind<ForOption, (A, B)> {
+    public static func product<A, B>(_ a: Kind<ForOption, A>, _ b: Kind<ForOption, B>) -> Kind<ForOption, (A, B)> {
         ForOption.zip(a, b)
-	}
+    }
+}
+
+// MARK: Instance of `Monoidal` for `Option`,
+extension ForOption: Monoidal {
```

Comment:
Should any of these be made public? 🤔

Reply:
Not necessary and I even think it is a compilation error. They are public written as it is.